front/js/components/order.es6

```diff
@@ -159,14 +154,11 @@
    * Return hash with customer's info from form.
    */
   const getOrderInfo = () => {
-    const orderInfo = {
-      payment_type: getSelectedPayment(),
-    };
-
-    $.each(DOM.orderForm, (name, field) => {
-      orderInfo[name] = $(field).val();
-    });
-
+    const orderInfo = Object.keys(DOM.orderForm).reduce((acc, key) => {
+      acc[key] = $(DOM.orderForm[key]).val();
```

Comment:
it seems to me, that map is more simple alternative:
```
Object.keys(DOM.orderForm).reduce((key) => {
return key: $(DOM.orderForm[key]).val();
}
```

Maybe syntax is not fully working. It's my not good js

Reply:
`.map(...)` will produce array of objects `[{a: b}, {b: c}, ...]` instead of `{a: b, b:c, ...}`